src/database/ProductExports/index.ts

```diff
@@ -0,0 +1,83 @@
+import dbConnection, { tableNames, execute } from '../connection';
+
+const prodExportTable = () => dbConnection()(tableNames.PRODUCT_EXPORTS);
+
+/**
+ * Represent a product export after retrieval from the database
+ */
+export interface ProdExportDb {
+  productexportuuid: string;
+  transportid: string;
+  datetime: string;
+  productname: string;
+  producttypeid: number;
+  amountofproduct: number;
+  lastmodified: string;
+}
+
+/**
+ * Represent a product export used in an INSERT call on the database
+ */
+export interface ProdExportDbInsertReq {
+  productexportuuid: string;
+  transportid: string;
+  datetime: string;
+  producttypeid: number;
+  amountofproduct: number;
+  lastmodified: string;
+}
+
+const builders = {
+  /** Get all product transactions of a certain type */
+  getSingleProdExport(productexportuuid: string) {
+    return prodExportTable().select('*')
```

Comment:
does this need to return the product name? or is the id enough?

Reply:
we should probably be consistent since the get all product exports api returns the productname too

Reply:
Good catch